solutions/mint-nft/README.md

```diff
@@ -0,0 +1,41 @@
+# mint-nft example
+
+This example shows how -------------------------
+
+## Demo
+
+https://mint-nft.vercel.app
```

Comment:
This demo link doesn't seem to be the one for this example

Reply:
@gbibeaul

Reply:
https://github.com/vercel/examples/pull/152/files#diff-c19ce76276cdaf4483d6038fa8143b1dff3b9a1390b3ec30e7ce0b1b266c7dbfL3